competencies/roles/DataScienceLevelMapping.md

```diff
@@ -0,0 +1,23 @@
+The three categories of competencies can be used to map to typical job titles specific roles. 
+
+This framework also uses the [Developer](./Developer.md)  competencies of 
+"Tech Design, Implementation and Code Ownership" and of "System Design" to create 
+a role family called Machine Learning Engineer
+
+
+| Title/Job                   | Key Competenies           | Other Competencies           |     Notes  |
+| --------------------------- |:-------------------------:| ----------------------------:| ----------:|
+| Data Science I (junior)     |                           | 90% emerging; 10% proficient |  New Grad, or first one or two years |
+| Data Science II             | Proficient: "Machine Learning Analysis"      |  50% emerging, 50% proficient  |   |
+| Data Science III a (senior) | Proficient: "Error, Causality, Inference" | 60%% proficient; some authority  |  | 
```

Comment:
Discrepancy here compared to chart below (60% proficiency here, but 50% below for Data Science III a)